WeScan/Review/ReviewViewController.swift

```diff
@@ -45,19 +54,47 @@ final class ReviewViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        enhancedImageIsAvailable = results.enhancedImage != nil
+        
         setupViews()
+        setupToolbar()
         setupConstraints()
         
-        title = NSLocalizedString("wescan.review.title", tableName: nil, bundle: Bundle(for: ReviewViewController.self), value: "Review", comment: "The review title of the ReviewController")
+        title = nil
```

Comment:
We should still display the title "Review" for now. This isn't ideal I agree, but it's better than not having a title at all.
This whole flow is going to change with the introduction of batch scanning, so it's only temporary.

Reply:
Fair enough, will do.

Reply:
Thanks!